src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -689,4 +704,15 @@ private Loan changeDueDate(DateTime dueDate, Loan loan) {
       super(new JsonObject().put("id", id));
     }
   }
+
+  public Integer getItemLimit() {
+    if (getLoansPolicy() != null) {
+      return getLoansPolicy().getInteger("itemLimit");
+    }
+    return null;
+  }
```

Comment:
Can we be sure that the second call to getLoansPolicy() will not return null if the first call does not return null? Now and in future? To avoid any analysis simply make one call only and store the result: `final JsonObject loansPolicy = getLoansPolicy();`

Using a guard clause for the null case makes it more easy to understand:

```
if (loansPolicy == null) {
  return null;
}
return loansPolicy.getInteger("itemLimit");
```

Reply:
Thanks! 
fixed